src/js/vendor/docsearch.bundle.js

```diff
@@ -13,6 +16,12 @@
     var searchFieldSelector = '#' + (config.searchFieldId || 'search')
     var searchField = document.querySelector(searchFieldSelector)
     var filterInput = searchField.querySelector('.filter input')
+    if (filterInput) {
+      var facetFilters = localStorage.getItem(docsearchFacetFiltersKey)
+      if (facetFilters && facetFilters.split(',').includes(filterInput.dataset.facetFilter)) {
```

Comment:
Do we support IE? According to MDN `Array.includes` is not supported on IE: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes#browser_compatibility

Reply:
We don't support IE, but I tend to still avoid includes just because I've gotten so used to doing so.